Sources/xcproj/PBXFileReference.swift

```diff
@@ -52,7 +52,7 @@ public class PBXFileReference: PBXObject, Hashable {
                 xcLanguageSpecificationIdentifier: String? = nil) {
         self.fileEncoding = fileEncoding
         self.explicitFileType = explicitFileType
-        self.lastKnownFileType = lastKnownFileType ?? path.flatMap { PBXFileReference.fileType(path: Path($0)) }
+        self.lastKnownFileType = lastKnownFileType
```

Comment:
What are your thoughts on this @yonaskolb?

Reply:
I guess if that' really the case. I can see why removing this might be good for other reasons as well, where we might be calculating the wrong `lastKnownFileType` as `PBXFileReference.fileType(` would never be perfect